scripts/novaseq/demux-novaseq.bash

```diff
@@ -61,6 +61,7 @@ for PROJECT_DIR in ${OUT_DIR}/${UNALIGNED_DIR}/*; do
     if [[ ${PROJECT} == 'Stats' ]]; then continue; fi
     if [[ ${PROJECT} == 'Reports' ]]; then continue; fi
     if [[ ${PROJECT} =~ Project_* ]]; then continue; fi
+    if [[ ${PROJECT} == 'indexcheck' ]]; then mv ${PROJECT_DIR} ${OUT_DIR}/${UNALIGNED_DIR}/Project_${PROJECT}; fi
```

Comment:
I'd say there is no reason to write this on the same line and would break it up into three lines for readability. What do you say?

Reply:
Agreed! Would it look like this?
```suggestion
    if [[ ${PROJECT} == 'indexcheck' ]]; then 
        mv ${PROJECT_DIR} ${OUT_DIR}/${UNALIGNED_DIR}/Project_${PROJECT}
    fi
```